arches/settings.py

```diff
@@ -529,6 +529,7 @@
 COPYRIGHT_TEXT = "All Rights Reserved."
 COPYRIGHT_YEAR = "2016"
 
+
```

Comment:
Could you remove this new line to keep the settings.py file out of the change

Reply:
Done